src/Compilers/CSharp/Test/Semantic/SourceGeneration/GeneratorDriverTests.cs

```diff
@@ -348,7 +348,8 @@ class C { }
             // now try apply edits
             driver = driver.TryApplyEdits(compilation, out editedCompilation, out succeeded);
             Assert.True(succeeded);
-            Assert.Equal(6, editedCompilation.SyntaxTrees.Count());
+            // PROTOTYPE: we get five, because the previous edit was forgotten in some cases
```

Comment:
Do we need to add anything tracking this? Or is // PROTOTYPE: fine?

Reply:
Oops that slipped back in. Good catch, thanks!